packages/plugins/blocks/blocks-antd/src/blocks/PhoneNumberInput/schema.json

```diff
@@ -127,6 +127,23 @@
           "displayType": "yaml"
         }
       },
+      "regex": {
```

Comment:
Think the field name can be more descriptive. Perhaps make it `replace`, with  `replace.pattern`, `replace.flags`, `replace.replacement`. options?

Reply:
Change made in https://github.com/lowdefy/lowdefy/pull/1579/commits/cd811ae6fae47a8384c73f439fd8fab86d0226ac